### PR TITLE
bump version in deployment docs

### PR DESCRIPTION
**Pull request by toidiu**
**IMPORTANT: Please do not create a Pull Request adding a new feature without discussing it first.**

The place to discuss new features is the forum: <https://zola.discourse.group/>
If you want to add a new feature, please open a thread there first in the feature requests section.

Sanity check:

* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

## Code changes
(Delete or ignore this section for documentation changes)

* [x] Are you doing the PR on the `next` branch?

If the change is a new feature or adding to/changing an existing one:

* [ ] Have you created/updated the relevant documentation page(s)?

### CODE REVIEW

**Comment by toidiu** [comment]
I think the base for this PR should be master since this is a doc change and this pr is updating to reflect the current state in master? Please correct me if I am mistaken or if this breaks other things.

File: docs/content/themes/even/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ created = 2019-04-06T11:27:43+02:00
 updated = 2018-01-25T18:44:44+01:00
 repository = "https://github.com/getzola/even"
 homepage = "https://github.com/getzola/even"
-minimum_version = "0.6.0"
+minimum_version = "0.7.0"
```
Comment (Keats) [inline docs/content/themes/even/index.md]: you can remove this change, it's autogenerated and doesn't need to be bumped

**Comment by Keats** [comment]
It's fine on master! I definitely need to have some checklist before releasing to avoid missing those all the time

**Comment by Keats** [comment]
Thanks!

**Pull request merged**